src/west/manifest.py

```diff
@@ -401,11 +403,12 @@ class Project:
 
     Projects are neither comparable nor hashable.'''
 
-    __slots__ = ('name remote url path abspath posixpath clone_depth '
+    __slots__ = ('name remote url path west_top abspath posixpath clone_depth '
```

Comment:
Just a nit, but `__slots__` is ordered, so would you mind not putting this in between the three xxpath fields? E.g. putting it at the end of the list?

Reply:
I put `west_stop` last here but I had to put it first in the ManifestProject() constructor to make it mandatory. Feedback welcome!

Reply:
> Feedback welcome!

I am not a fan of treating positionals as kwargs (as in `ManifestProject(west_top=foo, ....)` instead of `ManifestProject(foo, ...)`). The formal parameter names of positionals should not be considered API, in my opinion. If https://www.python.org/dev/peps/pep-0570/ were available in Python 3.4 I would use it everywhere.

But this is fine, just a personal maintainer quirk :).